src/megameklab/com/MegaMekLab.java

```diff
@@ -126,9 +129,12 @@ private static void showInfo() {
     }
     
     private static void startup() {
-        System.out.println("Starting MegaMekLab version: " + MegaMekLab.VERSION);
```

Comment:
This is already logged by showInfo(), which I moved earlier so it would run before initializing anything else.